app/assets/javascripts/client/common_list/EntryCtrl.js

```diff
@@ -0,0 +1,116 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('EntryCtrl', EntryCtrl);
+
+  EntryCtrl.$inject = [
+    '$location',
+    'SessionService'
+  ];
+
+  function EntryCtrl($location, SessionService) {
+    var vm = this;
+
+    vm.consensusReportIcon = function(entity) {
+      if (entity.consensus && entity.consensus.is_completed) {
+        return 'fa-check';
+      } else {
+        return 'fa-spinner';
+      }
+    };
+
+    vm.draftStatusIcon = function(entity) {
+      if (entity.has_access) {
+        return 'fa-eye';
+      } else {
+        return 'fa-minus-circle';
+      }
+    };
+
+    vm.roundNumber = function(number) {
+      return Math.floor(number);
+    };
+
+    vm.isNetworkPartner = function() {
+      return SessionService.isNetworkPartner();
+    };
+
+    vm.meetingTime = function(date) {
+      if (date != null) {
+        return moment(date, 'YYYY-MM-DDTHH:mm:ss').format('Do MMM YYYY');
+      } else {
+        return 'TBD';
+      }
+    };
+
+    vm.gotoLocation = function(location) {
+      if (location) {
+        $location.url(location);
+      }
+    };
+
+
+    vm.activeAssessmentLink = function(entity) {
+      if (vm.responseLink(entity)) {
+        return 'active';
+      } else {
+        return '';
+      }
+    };
+
+    vm.responseLink = function(entity) {
+      switch (entity.response_link) {
+        case 'new_response':
+          return '/assessments/' + entity.id + '/responses';
+        case 'response':
+          return '/assessments/' + entity.id + '/responses/' + entity.responses[0].id;
+        case 'consensus':
+          return '/assessments/' + entity.id + '/consensus/' + entity.consensus.id;
```

Comment:
None of it for now.  That was a straight-up port from the existing code.  In the future I anticipate that it will have a lot to do with it, but for now it doesn't.